content/apprenticeships/Flatiron.md

```diff
@@ -0,0 +1,9 @@
+---
+company: "Flatiron Health"
+description: "In this role, you'll work with a software engineering team to help oncologists spend more time with patients or empower scientists to conduct research faster."
+image: "/images/apprenticeships/flatiron.jpeg"
```

Comment:
Can you update this `image` line to:

```
image: "flatiron.jpg" 
```

It seems I need to update the documentation as adding a path to the image isn't valid, but just the image name and extension is valid.

Also you had an "e" in your file extension when you have the file saved as `.jpg`